libraries/Microsoft.Bot.Builder.Azure/CosmosDbStorage.cs

```diff
@@ -363,6 +359,30 @@ private async Task InitializeAsync()
             }
         }
 
+        private byte[] Serialize(object data)
```

Comment:
We can't take any of these changes - anyone running the code with data stored in CosmosDB would be broken. 

Also, the root behavior of serialize json, then zip it, and write it to CosmosDB seems flawed. 

You can easily author your own storage provider if that's the desired behavior.